Source/Core/VideoCommon/TextureConversionShader.cpp

```diff
@@ -681,6 +683,31 @@ static void WriteZ24Encoder(char*& p, APIType ApiType, const EFBCopyFormat& form
   WriteEncoderEnd(p);
 }
 
+static void WriteXFBEncoder(char*& p, APIType ApiType, const EFBCopyFormat& format)
+{
+  WriteSwizzler(p, GX_CTF_XFB, ApiType);
```

Comment:
Yeah, while I get your point and this would work, it feels really unfortunate that we have to mess with the texture coordinates to get a texture of the right size but cutting off the excess that we scaled.

I was hoping to do it using the viewport or the render region.  But then I wondered if it'd be easier if the code was a bit simpler and that is what led to #5749 .

If we can't do any of those things or we don't want to do those things, I'll continue with modifying the swizzler.  But as a newbie trying to learn the code, my first intuition was that it felt like the wrong place to put it.